eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -215,7 +216,7 @@ jobs:
       value: >-
         -subset installer -ci
         $(BuildAction)
-        /p:CrossBuild=${{ ne(parameters.crossrootfsDir, '') }}
+        /p:CrossBuild=${{ or(ne(parameters.crossrootfsDir, ''), and(eq(parameters.osGroup, 'OSX'), eq(parameters.archType, 'arm64'))) }}
```

Comment:
Hmm, there is a lot of duplication around this condition. I wonder if we should introduce a common variable or something to control this in `xplat-setup`. Not as part of this PR as it would complicate it somewhat, but might be worth exploring that idea.

Reply:
It seems that introducing a new `crossBuild` parameter instead of detecting cross build using `crossrootfsDir` or OSX arm64 would be the cleanest solution. In some follow up PR.

Reply:
Yeah I agree, that would be best.

Reply:
I'll generate the later PR when this merges